frontend/package.json

```diff
@@ -13,8 +16,15 @@
     "webfontloader": "^1.0.0"
```

Comment:
I believe you need to add asyncComputed to here unless it is included with vue3?